contracts/Reputation.sol

```diff
@@ -20,8 +20,10 @@ contract Reputation is Ownable {
         return reputation[_owner];
     }
 
-	function set_reputation(address _account, uint256 _amount) onlyOwner {
+	function setReputation(address _account, uint256 _amount) onlyOwner {
+	    uint prevTotalReputation = totalReputation; 
         totalReputation = totalReputation - reputation[_account] + _amount;
+        if( ( _amount >= reputation[_account] ) && ( prevTotalReputation > totalReputation ) ) throw; // overflow 
```

Comment:
I think if would be clearer that this code si doing what it needs to do if we we use the "safeAdd" from teh "safeMath" contract. So line 25 would become `totalRepuation = safeAdd(totalReputation - reputation[_account], _amount)`